cmake/onnxruntime.cmake

```diff
@@ -72,6 +73,7 @@ if (NOT WIN32)
         INSTALL_RPATH_USE_LINK_PATH FALSE
         BUILD_WITH_INSTALL_NAME_DIR TRUE
         INSTALL_NAME_DIR @rpath)
+      set(ONNXRUNTIME_SO_LINK_FLAG " -Wl,-exported_symbols_list,${SYMBOL_FILE}")
     else()
```

Comment:
This can be moved above with other ONNXRUNTIME_SO_LINK_FLAG setting together
Same above for OUTPUT_STYLE
But can be done after 1.5

Reply:
Yes, should be applicable for other MacOS/iOS, but due to the r1.5, it was limited on iOS.